example_code/basic_examples/sending_ether/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "stylus-sending-ether"
+version = "0.1.7"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/OffchainLabs/stylus-hello-world"
```

Comment:
Thanks! One more thing here, should we change the repository url here?

Reply:
Decided to remove the license from here, cause we already have it on another folder.